packages/components/organisms/f-content-cards/src/components/templates/ContentCardVoucherCode.vue

```diff
@@ -0,0 +1,201 @@
+<template>
+    <button
+        type="button"
+        :class="$style['c-contentCard-voucher']"
+        :data-test-id="`contentCard-${cardType}`"
+        @click="copyVoucherCode">
+        <span
+            :class="$style['c-contentCard-voucher-code']"
+            :data-test-id="`contentCard-${cardType}-code`">
+            {{ code }}
+        </span>
+        <span
+            :class="voucherCopyClasses"
```

Comment:
Any reason you haven't used the dynamic class approach here? e.g.

```html
:class="[$style['c-contentCard-voucher-copy'], {
    this.copyState === COPY_STATE_COOLDOWN: $style['c-contentCard-voucher-copy-copied'],
    this.copyState === COPY_STATE_TRANSITIONOUT: $style['c-contentCard-voucher-copy-transition-out']
}]"
```

Reply:
Not sure, it was copied across from the previous component. I agree it would simplify the code and make it more readable. I will make the change though. Thanks

Reply:
@DamianMullins I think this is because with `$style` you can't write `{$style['example-class']: this.copyState === COPY_STATE_COOLDOWN}` which would be the shorthand instead of using the ternary expression

Reply:
Ah yeah, sorry, my code above was completely wrong! You can apply conditional CSS module classes like this (we do this [in the mega-modal component](https://github.com/justeat/fozzie-components/blob/679296a56d83e49484146f4d9063c6c3bea7ca48/packages/components/molecules/f-mega-modal/src/components/MegaModal.vue#L14-L19).

```
:class="[$style['c-contentCard-voucher-copy'], {
    [$style['c-contentCard-voucher-copy-copied']]: this.copyState === COPY_STATE_COOLDOWN: ,
    [$style['c-contentCard-voucher-copy-transition-out']]: this.copyState === COPY_STATE_TRANSITIONOUT
}]"
```

Reply:
nice thanks that looks quite neat and tidy